examples/oft-solana/README.md

```diff
@@ -243,18 +259,29 @@ npx hardhat lz:oapp:wire --oapp-config layerzero.config.ts --solana-secret-key <
 
 With a squads multisig, you can simply append the `--multisigKey` flag to the end of the above command.
 
-### Send SOL -> Sepolia
+### Set Message Execution Options
```

Comment:
Can you reference enforcedOptions here and explain you can either set enforced options in the layerzero.config.ts (RECOMMENDED) or pass only extraOptions as the caller?

Reply:
added